app/presenters/tree_builder_chargeback_rates.rb

```diff
@@ -35,6 +35,24 @@ def x_get_tree_roots(count_only, options)
         end
         objects
       end
+    when :cb_rates
+      # the rate accordion merge the compute rate and the storage rate in one unic rate grouping by description
+      rates = ChargebackRate.all
+      grouped_rates = rates.group_by(&:description)
+      if count_only
+        grouped_rates.length
+      else
+        objects = []
+        grouped_rates.sort.map do |description_group|
+          objects.push(
+            # We identified by the structure Compute_id:Storage_id
+            :id    => description_group[1][0].id.to_s + ":" + description_group[1][1].id.to_s,
```

Comment:
@h-kataria Yes. This error is because you do not have rate pairs with the same name. I thought this PR thinking about the existence of pairs of rates (Compute and Storage) with the same name, or a intalacion to with the data from chargeback_rate.yml that have a compute rate and storage rate called "default ".
The existence of previous rates that do not have this feature would be thought later in another PR.